data/transport missions.txt

```diff
@@ -3469,3 +3469,114 @@ mission "Inheritance Redirection"
+		ship "Hauler III" "SB-Labs Testament"
+	on offer
+		conversation
+			`An elegant-looking lady approaches you.`
+			`	"Captain <last>, Captain <last>! It is such an honor to meet you!"`
+			choice
+				`	"Wow, that's quite a greeting. Why is it an honor to meet me?"`
+				`	"Hello, nice to meet you too."`
+					goto "hello too"
+				`	"Sorry, I don't have time to talk. I'm really busy."`
+					defer
+			`	She smiles and replies, "Well, I actually don't know you, but I just read your name on one of the cargo manifests. And my experience is that this greeting works best if you need something from a spaceship captain."`
+			label "hello too"
+			`She says, "My name is Vanessa Oxygenpocket and I am part of a team that is designing the new Hauler VI transport ship series here at Southbound Shipyards."`
+			choice
+				`	"'Oxygenpocket' is an uncommon name."`
+				`	"And what help do you need from me?"`
+					goto "help request"
+			`	Vanessa looks surprised and then responds, "Well, it might sound uncommon to people from a planet with a breathable atmosphere, but it is actually similar to Vanderberg, which means 'from the mountain', or Langley, which means 'woodland'.`
+			`	"Oxygenpockets were as much a feature on my home-planet as mountains and woodlands are on more habitable planets."`
+			label "help request"
+			`	Vanessa pauses for a moment and then continues, "We're testing the VI's cargo systems using a Hauler III. Normally our own department would escort such prototypes, but there is a mole in our organization and we have very strong competition from Syndicated Shipyards. I need you to help us travel to <destination> to deliver sushi for a formal dinner. You will be paid for your expenses. Will you help us?"`
+			choice
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on visit
+		dialog `You land on <destination>, but the Hauler prototype didn't catch up with you yet.`
+	on complete
+		dialog
+			`You are waiting for the Hauler prototype to land, but it remains in orbit.`
+			`There is no response to your hails by radio. You need to take off and investigate.`
+
+mission "Hauler VI cargo prototype 2"
+	name "Escort Hauler prototype to student party"
+	description "Escort the Hauler prototype to <destination> to deliver cooked fish dishes to a streetfood festival."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 1: done"	
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel, "Why were you trying to contact me over radio! Does everyone need to know that we are flying around here?"`
+			`She adds, "The cooling failed and the sushi nearly cooked, let's go to <destination> to deliver it as fish dishes to a streetfood festival."`
```

Comment:
```suggestion
			`She adds, "The cooling failed and the sushi nearly cooked. Let's go to <destination> to deliver it as fish dishes to a streetfood festival."`
```

What was the point of the mission again? Was is to be an escort? Or was it to deliver the cargo? Because if it's the former, then the food doesn't matter, and if it's the latter, then they just failed and nonchalantly switching destinations is very odd.

This whole thing with delivering food just feels very strange to me overall.

Reply:
The point of the mission for Vanessa is that she wants to show off how well the new cargo refrigeration systems for the Hauler VI are working (by transporting food for an important dinner and thereby showing at the dinner how well the food is preserved during transport).

Vanessas showing off failed, since the cooled products end up being completely burned.

The mission for the player was to just escort the ship, the escorting part succeeds, since the player just accompanies the ship to it's last destination.

Reply:
All of that would be great to communicate in the mission itself so players don't need to ask you. ;-)

Reply:
Yes, having the story clear in mind and having the story properly written down are two different things. :-) Thanks for pointing out the gaps.